noxfile.py

```diff
@@ -28,9 +28,35 @@
     "cryptography",
     "responses",
     "grpcio",
+    "pytest-asyncio",
+    "aioresponses",
 ]
+
+TEST_DEPENDENCIES2 = [
```

Comment:
can we share this list, composing `test_dependencies` from this list and adding the other two?

Reply:
Fixed in the latest commit to only have the 1 TEST_DEPENDENCIES list.